client/my-sites/upgrades/components/form/country-select.jsx

```diff
@@ -57,6 +56,8 @@ export default React.createClass( {
 			} ) );
 		}
 
+		value = value || '';
```

Comment:
I think it might be better to do this immediately after you set the initial value in the let statement. Otherwise this is floating on its own, not close to its assignment or its use.